requirements.txt

```diff
@@ -1,5 +1,6 @@
 cwltool==1.0.20180912090223
 future>=0.16.0
 requests>=2.18.2
-py-tes>=0.2.1
+PyJWT
+-e git+git@github.com:CSCfi/py-tes.git@oauth2_token#egg=tes
```

Comment:
I would recommend to pin the dependencies to a version or commit:

PyJWT==1.6.4
-e git+git@github.com:ohsu-comp-bio/py-tes.gitt@199604418f06d72fd5fd200ed67f3cc223d4544a#egg=tes